priv/repo/migrations/20220208162404_wines_add_winery_id_column.exs

```diff
@@ -0,0 +1,11 @@
+defmodule WineOInventoryApi.Repo.Migrations.WinesAddWineryIdColumn do
+  use Ecto.Migration
+
+  def change do
+    alter table("wines") do
+      add :winery_id, references(:wineries)
```

Comment:
So if wine can exist without a winery then if the winery is deleted, the wine should remain, but what happens to the `winery_id` on `wines` in this case?

Reply:
The `on_delete` option has a `nilify_all` action. From my understanding, if the parent, which in this case is the `Winery` is deleted, then the `winery_id` in the `Wines` table associated with that record will be set to `nil`. Should I go ahead and make that change, or should we discuss more during our 1-1?

Reply:
After our 1-1 Sunny and I agreed that this is probably fine as he gets a foreign key constraint exception when he tries to do the delete. Safer would be to use the `nilify_all` although it might be a little overkill

Reply:
I ended up adding the `nilify_all` just to be safe!